src/main/webapp/app/components/sidebar/NavigationSidebar.tsx

```diff
@@ -1,76 +1,236 @@
-            <MenuItem icon={<b style={{ fontSize: '1.5em' }}>A</b>}>
-              Alteration <NavLink to={PAGE_ROUTE.ALTERATION} />
-            </MenuItem>
-            <MenuItem icon={<FontAwesomeIcon size="lg" icon={faFileAlt} />}>
-              Article <NavLink to={PAGE_ROUTE.ARTICLE} />
-            </MenuItem>
-            <MenuItem icon={<FontAwesomeIcon size="lg" icon={faPills} />}>
-              Drug <NavLink to={PAGE_ROUTE.DRUG} />
-            </MenuItem>
-            <MenuItem icon={<FontAwesomeIcon size="lg" icon={faBuilding} />}>
-              CDx
-              <NavLink to={PAGE_ROUTE.CDX} />
-            </MenuItem>
-            <MenuItem icon={<b style={{ fontSize: '1.2em' }}>CDX</b>}>
-              Specimen Type
-              <NavLink to={PAGE_ROUTE.SPECIMEN_TYPE} />
-            </MenuItem>
-            <MenuItem icon={<b style={{ fontSize: '1.2em' }}>FDA</b>}>
-              Submission <NavLink to={PAGE_ROUTE.FDA_SUBMISSION} />
-            </MenuItem>
-            <MenuItem icon={<b style={{ fontSize: '1.2em' }}>CT</b>}>
-              Condition <NavLink to={PAGE_ROUTE.CT_GOV_CONDITION} />
-            </MenuItem>
-            <MenuItem icon={<b style={{ fontSize: '1.2em' }}>EG</b>}>
-              Ensembl Gene <NavLink to={PAGE_ROUTE.ENSEMBL_GENE} />
-            </MenuItem>
-            <MenuItem icon={<b style={{ fontSize: '1.2em' }}>T</b>}>
-              Transcript <NavLink to={PAGE_ROUTE.TRANSCRIPT} />
-            </MenuItem>
+            <MenuItemCollapsible
+              isCollapsed={props.isNavSidebarCollapsed}
+              text="Curation"
+              icon={<FiFileText size={25} />}
+              nav={<NavLink to={PAGE_ROUTE.CURATION} />}
+            />
+            <MenuItemCollapsible
+              isCollapsed={props.isNavSidebarCollapsed}
+              text={'Search'}
+              icon={<BiSearchAlt size={25} />}
+              nav={<NavLink to={PAGE_ROUTE.SEARCH} />}
+            />
+            <SubMenu defaultOpen label="Entities" icon={<GoDatabase size={25} />}>
+              {entityMenuOrder.map(entityType => (
+                <PriorityEntityMenuItem key={entityType} type={entityType} handlePriorityMenuItemClick={handlePriorityMenuItemClick}>
+                  {ENTITY_MENU_NAME[entityType]}
+                </PriorityEntityMenuItem>
+              ))}
+            </SubMenu>
           </Menu>
-        </SidebarContent>
-      </ProSidebar>
-    </div>
+        </div>
+        <MenuDivider />
+        <Menu>
+          <SubMenu label={props.account.firstName} icon={<FaUserCircle size={25} />}>
+            <MenuItem component={<NavLink to={PAGE_ROUTE.ACCOUNT} />}>Account Settings</MenuItem>
+            <MenuItem component={<NavLink to={PAGE_ROUTE.ADMIN_USER_MANAGEMENT} />}>User Management</MenuItem>
+          </SubMenu>
+          <MenuItemCollapsible
+            isCollapsed={props.isNavSidebarCollapsed}
+            text={'Sign out'}
+            icon={<FaSignOutAlt size={25} />}
```

Comment:
Worth to create a constant for the fontsize so it could be used for all within the nav.